modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/Sort.js

```diff
@@ -20,11 +20,15 @@ export default function ({namespace: portletNamespace}) {
 		).value;
 
 		if (key) {
-			document.location.search = FacetUtil.updateQueryString(
+			const url = new URL(window.location.href);
+
+			url.search = FacetUtil.updateQueryString(
 				key,
 				[sortSelect],
 				document.location.search
```

Comment:
window.location

Reply:
Thanks, understandably better to use `window.location`! Just to make sure, would it be `new URL(window.location.href)` or `new URL(window.location)`?